functests/1_performance/netqueues.go

```diff
@@ -83,15 +83,15 @@ var _ = Describe("[ref_id: 40307][pao]Resizing Network Queues", func() {
 
 	Context("Updating performance profile for netqueues", func() {
 		It("[test_id:40308][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] Network device queues Should be set to the profile's reserved CPUs count ", func() {
-			devices := make(map[string][]int)
+			nodesDevices := make(map[string]map[string]int)
```

Comment:
please copy/paste the PR description in the commit message so we don't depend on github (but rather on git history) for the rationale of this change